src/test/java/hw6/steps/ThenSteps.java

```diff
@@ -0,0 +1,89 @@
+package hw6.steps;
+
+import data.UserTableRow;
+import hw3.pages.DifferentElementsPage;
+import hw3.pages.UserTablePage;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.Then;
+import utils.CalculationUtils;
+import utils.WebDriverSingleton;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class ThenSteps {
+
+    private DifferentElementsPage differentElementsPage;
+    private UserTablePage userTablePage;
+
+    @Then("For each checkboxes {string} and {string} there is an individual log row and value is corresponded to the status of checkbox")
```

Comment:
How will you manage if you will need add one more checkbox?

Reply:
added list